client/utils/auth.py

```diff
@@ -20,7 +20,10 @@
 CLIENT_ID = 'ok-client'
 # The client secret in an installed application isn't a secret.
 # See: https://developers.google.com/accounts/docs/OAuth2InstalledApp
-CLIENT_SECRET = 'EWKtcCp5nICeYgVyCPypjs3aLORqQ3H'
+# However, for other authentication providers such as Azure Active Directory
+# this might not be the case
+CLIENT_SECRET = os.getenv('OK_CLIENT_SECRET',
```

Comment:
Could you make clear that this is the secret for the ok-client app on the ok service. I think that was unclear before. People can find the secret value at `example.com/admin/clients/ok-client`?

Reply:
Done in d3a614a.